test/elixir_google_scraper_web/features/upload_keywords/upload_keywords_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule ElixirGoogleScraperWeb.UploadKeywords.UploadKeywordsTest do
-  use ElixirGoogleScraperWeb.FeatureCase, async: true
+  use ElixirGoogleScraperWeb.FeatureCase
```

Comment:
Just wondering why we're removing `async: true` from all feature cases? 🤔

Reply:
I think we can leave it @junan, Later when we have time, we can test it again. Maybe you can create an issue.

Reply:
@rosle I found the issue(https://github.com/parroty/exvcr/issues/58, https://github.com/parroty/exvcr/issues/53), it is related to `Exvcr` package that it sometime can't run tests parallel which it used a VCR cassette. I created a ticket for it to check it more later https://github.com/junan/elixir_google_scraper/issues/39